templates/content/cards/table-card.jsx

```diff
@@ -53,7 +53,14 @@ const SidebarContent = observer(
             </DetailList.Item>
             {doi && <DetailList.Item name="DOI">{doi}</DetailList.Item>}
             <DetailList.Item name="Update date">
-              {dayjs(lastUpdate).format('Do MMMM YYYY, hh:mm:ss')}
+              {formatDate(lastUpdate, {
+                year: 'numeric',
+                month: 'long',
+                weekday: 'short',
+                hour: 'numeric',
+                minute: 'numeric',
+                second: 'numeric',
+              })}
```

Comment:
it can be simplified to `formatDate(lastUpdate)` I just wanted to keep old format.

Reply:
If you do so, you will not need to construct a new object per each formatting. When you have 100 rows with a date in each to format, it sounds like a significant improvement.

Reply:
I am not sure what you are implying to.

Reply:
Oh I see. DateTimeFormat can be cached...

Reply:
I would prefer to construct `new Intl.DateTimeFormat` outside the `formatDate` function to prevent constructing it on every call improving performance at the same time. 

However, if you do really want to have options or do not want to change anything, current state of the PR is fine too.